src/modules/news/news.js

```diff
@@ -1,126 +1,168 @@
+        return this._container;
     }
 
-    _createCard() {
-        const card = document.createElement('div');
-        card.setAttribute('class', 'news__card');
-        card.appendChild(this._createImg());
-        card.appendChild(this._createCardDetails());
-        return card;
+    _createCard(parameter) {
+        this._card = createElement('div', { class: 'news__card' }, null, null);
+        this._card.appendChild(this._createImg(parameter));
+        this._card.appendChild(this._createCardDetails(parameter));
+        return this._card;
     }
 
-    _createImg() {
-        const img = document.createElement('img');
-        this._setAttributes(img, {
-            src: 'https://images.unsplash.com/photo-1636884285351-0f14627ada21?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1170&q=80',
+    _createImg(parameter) {
+        this._img = createElement('img', {
             alt: '',
             class: 'news__img',
         });
-
-        return img;
+        if (parameter.urlToImage) {
+            this._img.setAttribute('src', `${parameter.urlToImage}`);
+        } else {
+            this._img.setAttribute(
+                'src',
+                'https://images.unsplash.com/photo-1476242906366-d8eb64c2f661?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1469&q=80'
+            );
+        }
+        return this._img;
     }
 
-    _createCardDetails() {
-        const cardDetails = document.createElement('div');
-        cardDetails.setAttribute('class', 'card__details');
-        cardDetails.appendChild(this._createTag());
-        cardDetails.appendChild(this._createArticleTitle());
-        cardDetails.appendChild(this._createArticleContent());
-        cardDetails.appendChild(this._createLink());
-        return cardDetails;
+    _createCardDetails(parameter) {
+        this._cardDetails = createElement('div', { class: 'card__details' });
+        this._cardDetails.appendChild(this._createDate(parameter));
+        this._cardDetails.appendChild(this._createArticleTitle(parameter));
+        this._cardDetails.appendChild(this._createArticleContent(parameter));
+        this._cardDetails.appendChild(this._createLink(parameter));
+        return this._cardDetails;
     }
-    _createTag() {
-        const tag = document.createElement('span');
-        tag.setAttribute('class', 'news__tag');
-        tag.innerText = 'tag name';
-        return tag;
+
+    _createDate(parameter) {
+        let data = parameter.publishedAt.slice(0, 10);
+        this._date = createElement('div', { class: 'news__date' }, null, `Published: ${data}`);
+        return this._date;
     }
 
-    _createArticleTitle() {
-        const title = document.createElement('div');
-        title.setAttribute('class', 'news__article__title');
-        title.innerText = 'Title';
-        return title;
+    _createArticleTitle(parameter) {
+        this._articleTitle = createElement(
```

Comment:
check all the code and replace this._something with just const something (only where it can be possible - we dont use them in global-class scope)

we dont need to have class properties with these things